src/dotnet/APIView/APIViewWeb/LeanModels/ReviewRevisionListModels.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace APIViewWeb.LeanModels
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum RevisionType
+    {
+        Manual = 0,
+        Automatic,
+        PullRequest,
+        All
+    }
+
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ReviewState
+    {
+        Open = 0,
+        Closed
+    }
+
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ReviewRevisionStatus
```

Comment:
Can this be changed to approval status so we can use it for both API approval at revision level as well as approval at review level to indicate approval for first preview release.